### ISSUE TITLE
Enhance setup scripts

### ISSUE THREAD

**Issue by nsrCodes** [nsrCodes/cdd#2]
1. Improve and cover edge cases in `install.sh`
2. Create an `uninstall.sh`

**Comment by koolskateguy89** [comment]
It only works with Bash (appending to .bashrc in the install script) - you might want to add that to the README

**Comment by c-nv-s** [comment]
A little bit of advice.
If you begin your scripts with `` #!/usr/bin/bash `` the script will fail unless the person using it specifically has their bash binary at the location /usr/bin/bash 
On some systems it could be at /bin/bash or someone might have it at /usr/local/bin/bash
To make your scripts more portable you can ask the ``env`` command to find and use the desired binary wherever it is on the user's system by starting your scripts with `` #!/usr/bin/env bash `` ( or if it was a python script for example `` #!/usr/bin/env python `` )

Also, if your script depends on a command ( ``xdg-open`` ) to already be available in the user's ``$PATH``, then somewhere early in your script you could do a little test for the command and exit the script if the command isn't available or just advise that the ``cdo`` command will not work without it and continue.
```
#!/usr/bin/env bash

...

 if ! xdg-open --help 2>/dev/null ; then { printf "The xdg-open command is not available in your \$PATH \\nIt is usually installed with the package xdg-utils \\n " ; exit 1 ; }  fi
```

**Comment by nsrCodes** [comment]
@c-nv-s  Thank you for proposing these changes. I do not have much experience programming in bash and so missed out on all this(*hoping to get better soon*). The changes have been added.
I used `command -v xdg-open &>/dev/null` instead of using the help prompt because in that case if the command did exist, it would show the help text in the terminal.

Also removed the exit statement and handled that with an `else`. Else, since the script  runs as source the terminal would close if  the command does not exist (*I think the correct explanation is that the terminal and the script are no longer different processes if we use `source`/`.`, hence if you `exit 1` out of the script, you exit out of the terminal too*)

Once again thanks for pointing this out. Helped me learn something new.
